.github/workflows/gradle.yml

```diff
@@ -52,7 +51,8 @@ jobs:
     runs-on: macos-latest
     strategy:
       matrix:
-        api-level: [ 28, 29 ]
+        api-level: [ 26, 28, 29 ]
```

Comment:
I don't know what the best combo here is... 🤷‍♀️

I added 26 because that's the minimum set in the Gradle files. Maybe 28 can be removed?

The max level set in Gradle is 32, but when I set it, it failed - something about the combo not being available? https://github.com/ReactiveCircus/android-emulator-runner/issues/234

Apparently 30+ is slow https://github.com/ReactiveCircus/android-emulator-runner/issues/222